### DISCUSSION TITLE
Support for modification of HIP Objects

### DISCUSSION THREAD

**Discussion by tohchentat** [PaloAltoNetworks/pan-os-ansible#216] [Ideas]
Requesting for support of modification of hip objects, example, removing existing patch version and updating it by adding new one. 

Example of current API call to add new missing patch member to Patch Management under HIP Object:

https://ip/api/?key=&type=config&action=set&xpath=/config/devices/entry[@name='localhost.localdomain']/vsys/entry[@name='vsys1']/profiles/hip-objects/entry[@name='patch-mgt']&element=<patch-management><criteria><missing-patches><patches><member>1267</member></patches></missing-patches></criteria></patch-management>

**Comment by mrichardson03** [comment]
Please see #219 for a way this function could be added without having to create a module to do it.
